upgrade_tests/test_existance_relations/cli/test_virtwho.py

```diff
@@ -0,0 +1,62 @@
+"""Upgrade TestSuite for validating Satellite virt-who-config existence post upgrade
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: virt-who-config
+
+:TestType: nonfunctional
+
+:CaseImportance: High
+
+:SubType1: installability
+
+:Upstream: No
+"""
+import pytest
+from upgrade_tests.helpers.common import existence
+from upgrade_tests.helpers.existence import compare_postupgrade, pytest_ids
+
+# Required Data
+component = 'virt-who-config'
+virtwho_name = compare_postupgrade(component, 'name')
+virtwho_interval = compare_postupgrade(component, 'interval')
+virtwho_status = compare_postupgrade(component, 'status')
+virtwho_last_report = compare_postupgrade(component, 'last report at')
```

Comment:
Remove this,  before merging, as its no more required.

Reply:
oh, forgot this, thanks:)